cli/tests/019_media_types.ts

```diff
@@ -2,14 +2,14 @@
 // based on the URL containing `.t#.` strings, which exercises the different
 // mapping of media types end to end.
 
-import { loaded as loadedTs1 } from "http://localhost:4545/cli/tests/subdir/mt_text_typescript.t1.ts";
-import { loaded as loadedTs2 } from "http://localhost:4545/cli/tests/subdir/mt_video_vdn.t2.ts";
-import { loaded as loadedTs3 } from "http://localhost:4545/cli/tests/subdir/mt_video_mp2t.t3.ts";
-import { loaded as loadedTs4 } from "http://localhost:4545/cli/tests/subdir/mt_application_x_typescript.t4.ts";
-import { loaded as loadedJs1 } from "http://localhost:4545/cli/tests/subdir/mt_text_javascript.j1.js";
 import { loaded as loadedJs2 } from "http://localhost:4545/cli/tests/subdir/mt_application_ecmascript.j2.js";
-import { loaded as loadedJs3 } from "http://localhost:4545/cli/tests/subdir/mt_text_ecmascript.j3.js";
 import { loaded as loadedJs4 } from "http://localhost:4545/cli/tests/subdir/mt_application_x_javascript.j4.js";
+import { loaded as loadedTs4 } from "http://localhost:4545/cli/tests/subdir/mt_application_x_typescript.t4.ts";
+import { loaded as loadedJs3 } from "http://localhost:4545/cli/tests/subdir/mt_text_ecmascript.j3.js";
+import { loaded as loadedJs1 } from "http://localhost:4545/cli/tests/subdir/mt_text_javascript.j1.js";
+import { loaded as loadedTs1 } from "http://localhost:4545/cli/tests/subdir/mt_text_typescript.t1.ts";
+import { loaded as loadedTs3 } from "http://localhost:4545/cli/tests/subdir/mt_video_mp2t.t3.ts";
+import { loaded as loadedTs2 } from "http://localhost:4545/cli/tests/subdir/mt_video_vdn.t2.ts";
```

Comment:
It sorts by module specifier (absolute first, then relative descending based on `../`), so for example:

```ts
import {} from "./a";
import {} from "../a";
import {} from "../../b";
import {} from "https://deno.land/b.ts";
import {} from "https://deno.land/a.ts";
import {} from "../../a";
```

Goes to...

```ts
import {} from "https://deno.land/a.ts";
import {} from "https://deno.land/b.ts";
import {} from "../../a.ts";
import {} from "../../b.ts";
import {} from "../a.ts";
import {} from "./a.ts";
```

The code here's isn't exactly the best example of when this might be desired.